### ISSUE TITLE
Advisory missing for CVE-2021-29425 - commons-io

### ISSUE THREAD

**Issue by janlaan** [OSSIndex/vulns#181]
**Advisory details**
```
  URL: https://nvd.nist.gov/vuln/detail/CVE-2021-29425
  format: maven
  namespace: commons-io
  name: commons-io
  versions: 2.2 through 2.6
```

**More information**

- https://ossindex.sonatype.org/component/pkg:maven/commons-io/commons-io

**Comment by ken-duck** [comment]
This has been resolved and should show up by sometime tomorrow.

**Comment by msymons** [comment]
@ken-duck, the fix is not yet showing up.

**Comment by savujevi** [comment]
Hello,
as shown in my link https://github.com/DependencyTrack/dependency-track/issues/1139 the fix seems not to shown up.

Thank you for your help.

**Comment by msymons** [comment]
@ken-duck, sorry to nag...  but it has now been over 6 weeks since this was resolved but the component is still reporting in OSS Index with no linkage to the CVE.

**Comment by ken-duck** [comment]
Many apologies for not resolving this properly. I have obviously fallen behind on tracking the resolution status of old issues. We are working on changing the research pipeline which should not only resolve these issues sooner, but also result in far fewer issues to begin with.

**Comment by msymons** [comment]
@ken-duck, the vulnerability is now correctly reporting...  but only for some versions.

The description says "In Apache Commons IO **before 2.7**" OSS Index only reports for 2.2, 2.3, 2.4, 2.5,& 2.6.

I have older occurrences that are not alerting.  For instance,  2.1 (which happens to still be a transitive dependency of the very latest Keycloak 16.1.1).

**Comment by ken-duck** [comment]
This past year among other changes we upgraded the OSS Index vulnerability database. The new database has significantly more vulnerabilities, is much more actively maintained, and has more in depth research on many of the issues. The issue reported here was resolved  in the new database and as such we are closing the issue.

For the record, we are migrating to a new email-based reporting system in order to better mesh with our internal processes, which will allow us to be more reactive to our users.
